### PR TITLE
Fix build.zig for 0.12.1, and add running instructions to readme, detect inkscape layer names

### PR DESCRIPTION
**Pull request by coryalder**
1. Add support for Inkscape SVGs. This is done by adding selectors for the `inkscape:labal` attribute, which inkscape uses instead of `id` (e.g. `[*|label=\"Drills\"]"`).
2. Fix builds - the github action was depending on 0.12.0-dev which is no longer available. I had to change `build.zig` to make it work with 0.12.1. Zig 0.13.0 is out now, but that needed more changes to make it work and I _barely_ understand what I'm doing with zig :)
3. Add running instructions to the readme, so it's easier to get up and running.

Feel free to reject any or all of this PR - I just wanted inkscape svgs to work, and it's working on my fork so I am happy - but I figured I'd share back the work I did. Thanks for this tool! :)

### CODE REVIEW

**Comment by coryalder** [comment]
Oops. Hold on the inkscape part of this - looks like it's not loading up files at the right size - the export is miiiiniscule!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ Gingerbread is a tool for converting vector artwork to KiCAD PCB files that live
 
 See https://gingerbread.wntr.dev for more information.
 
+## Building & Running
```
Comment (theacodes) [inline README.md:7]: Thank you so much for adding this. You're the best.

**Review by theacodes — APPROVED** [review]
This looks great, how would you feel about merging this as-is for the build improvements and doing a follow-up to fixup the inkscape scale issue?

**Comment by coryalder** [comment]
Yup, I think this is ok to merge! Doesn't solve all the issues, but it's a step in the right direction!

I managed to scale my images up in inkscape to get them rendering almost correct using this change. Seems like the drill layer didn't work tho. Hope to find some time to re-visit this soon!

**Comment by theacodes** [comment]
Huzzah! Thank you!

**Pull request merged**